whichone/static/js/main.js

```diff
@@ -7,6 +7,8 @@ var user = {
     scores: []
 };
 
+var muteSound = false;
+
```

Comment:
Should it be a persisted user setting (i.e. LocalStorage)

Reply:
done